draft-ietf-sedate-datetime-extended.md

```diff
@@ -367,6 +367,19 @@ practice of interpreting the local offset `Z`: this is no longer
 understood to "imply that UTC is the preferred reference point for the
 specified time".
 
+A revised {{Section 4.3 of RFC3339}} with the update could read as follows:
+
+{:quote}
+>
+   If the time in UTC is known, but the offset to local time is unknown,
+   this can be represented with an offset of "Z".
+   (The original version of this specification provided "-00:00" for
+   this purpose, which is not allowed by {{ISO8601-2000}} and therefore
+   is less interoperable; {{RFC2822}} describes a related
```

Comment:
RFC 2822 is obsoleted by RFC 5322; surely it would make more sense to reference that ?
Or does it not contain the same convention ?
I'd be better placed to discern that if you gave some hint to where in the RFC the convention is described.
In 4.3, the single-letter codes are described as having various offset meanings, with Z equivalent to +0000, but we are told they *should* be treated as -0000 unless out-of-band evidence says otherwise.
The same appears in 5322:
https://datatracker.ietf.org/doc/html/rfc5322#section-4.3

Reply:
The second paragraph of Section 2 already mentions that the convention is also continued in 5322, and I note that this is also true of 5322bis (https://www.ietf.org/archive/id/draft-ietf-emailcore-rfc5322bis-05.html#section-3.3-6).

We could change {{RFC2822}} into {{Section 3.3 of RFC5322}}.

Reply:
> .. and, of course, their references to -0000 implies that they, too, are in need of an update in light of the sedate work.

No, they don't have the problem, as their syntax is not related to ISO 8601.

Reply:
> we are told they _should_ be treated as -0000

Actually, the rationale only applies to A-Y, but the text essentially tells us the military values are non-interoperable.  But, as I said, the whole thing is irrelevant to us.

Reply:
> We could change {{RFC2822}} into {{Section 3.3 of RFC5322}}.

Note that the original text of Section 4.3 of RFC 3339 references RFC 2822.
We could of course update this to 5322 while we are adding the section reference.

Reply:
> > .. and, of course, their references to -0000 implies that they, too, are in need of an update in light of the sedate work.
> 
> No, they don't have the problem, as their syntax is not related to ISO 8601.

It remains that consistency among RFCs is surely desirable, especially when the RFCs in question plainly have been borrowing ideas off one another in the past.

Reply:
Consistency is certainly desirable, if it can be achieved (at a reasonable cost).
However, there is no chance that a change in 5322bis would see any traction.
It would simply increase uncertainty about what things actually mean.